app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -45,7 +45,10 @@ public Result getSelfParticipant() {
         
         StudyParticipant participant = participantService.getParticipant(study, session.getId(), false);
         
-        return okResult(participant);
+        // Do not return healthCode. This will shortly be handled with Jackson filters.
+        StudyParticipant updated = new StudyParticipant.Builder().copyOf(participant)
+                .withHealthCode(null).withEncryptedHealthCode(null).build();
```

Comment:
Is it necessary to clear out both health code and encrypted health code? They're both backed by the same member var. Or is this a "just in case" kind of thing, in case we need to change the implementation?